core/src/main/scala/kafka/server/epoch/LeaderEpochFileCache.scala

```diff
@@ -106,7 +106,7 @@ class LeaderEpochFileCache(topicPartition: TopicPartition,
       entry.epoch >= newEntry.epoch || entry.startOffset >= newEntry.startOffset
     }
 
-    if (removedEpochs.size > 1
+    if (removedEpochs.lengthCompare(1) > 0
```

Comment:
FYI reviewers -
removedEpochs is a scala Seq, hence, lengthCompare() is beneficial.